README.md

```diff
@@ -2,6 +2,8 @@
 
 **Capact** (pronounced: "cape-act", /ˈkeɪp.ækt/) is a simple way to manage applications and infrastructure.
 
+  > **⚠️ WARNING**: Capact versions prior to 0.4.0 are considered experimental. Capact 0.4.0, the very first public Capact release, is coming very soon. Stay tuned for the announcement!
```

Comment:
we can also add a short sentence that our website will be live after 0.4.0, for now you can read docs in our `capactio/website` repo and add a note with apologies for broken links 👍